doc/releases/quincy.rst

```diff
@@ -5,6 +5,593 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.7 Quincy
+==============
+
+This is the seventh backport release in the Quincy series. We recommend
+that all users update to this release.
+
+Notable Changes
+---------------
+
+* `ceph mgr dump` command now displays the name of the mgr module that
+  registered a RADOS client in the `name` field added to elements of the
+  `active_clients` array. Previously, only the address of a module's RADOS
+  client was shown in the `active_clients` array.
+
+* mClock Scheduler: The mClock scheduler (default scheduler in Quincy) has
+  undergone significant usability and design improvements to address the slow
+  backfill issue. Some important changes are:
+  * The 'balanced' profile is set as the default mClock profile because it
+    represents a compromise between prioritizing client IO or recovery IO. Users
```

Comment:
Sphinx says there is an unexpected indentation on this line.

Reply:
(line 26)

Reply:
This is caused because there isn't a blank line before line 25. I know that that might sound crazy, but that's Sphinx.

Reply:
Noted.. fixing

Reply:
What exactly has to be changed?

Reply:
@yuriw: I'll take care of it. Don't worry about anything.